docs/standard/microservices-architecture/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md

```diff
@@ -4,7 +4,7 @@ description: .NET Microservices Architecture for Containerized .NET Applications
 keywords: Docker, Microservices, ASP.NET, Container
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 05/26/2017
+ms.date: 10/30/2017
```

Comment:
This date represents how "fresh" the content is. It should either be now, in December, if you did a thorough review of the topic, or stay with the May date if you did not review it for accuracy.

Reply:
👍